### PR TITLE
Add lshw and update to latest kernel in this context

### PR DESCRIPTION
**Pull request by asafm-lb**
This is in preparation to send machine info (disks/ram/cpu) in context of checkin message

### CODE REVIEW

File: Makefile.build [hunk 1]
```diff
@@ -1,4 +1,4 @@
-KERNEL_VERSION = 3.10.0-327.36.3.el7.x86_64
+KERNEL_VERSION = 3.10.0-514.6.1.el7.x86_64
```
Comment (sashas-lb) [inline Makefile.build]: please split this into separate commit

File: docker/build-inaugurator.dockerfile [hunk 2]
```diff
@@ -33,6 +33,8 @@ RUN yum install -y \
     python-pip \
     make \
     kernel \
+    lshw \
+    vim \
```
Comment (sashas-lb) [inline docker/build-inaugurator.dockerfile]: why do you install vim into the docker ? please split it into different commit if you still need it

Comment (asafm-lb) [inline docker/build-inaugurator.dockerfile]: will remove vim...
